saliency/blur_ig_test.py

```diff
@@ -0,0 +1,55 @@
+"""Tests completeness axiom for blur_ig.
+"""
+
+from absl.testing import absltest
```

Comment:
use `googletest` for testing. See https://github.com/PAIR-code/saliency/blob/master/saliency/xrai_test.py#L5

Reply:
Done.

JFYI, I continue to use tensorflow.compat.v1 (instead of import tensorflow as tf to avoid warnings like: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead (similar warning for placeholder))
Also, it seems tensorflow.compat.v1 doesn't have the googletest.
`from tensorflow.compat.v1.python.platform import googletest
`
Gives  ImportError: No module named python.platform

In any case using
```
import tensorflow.compat.v1 as tf
from tensorflow.python.platform import googletest
```
resolves issues.